perun-core/src/main/java/cz/metacentrum/perun/core/bl/PerunBl.java

```diff
@@ -223,4 +223,10 @@
 	 * @return  ModulesUtilsBl
 	 */
 	ModulesUtilsBl getModulesUtilsBl();
+
+	/**
+	 * Gets a TasksManagerBl
+	 * @return TasksManagerBl
+	 */
+	TasksManagerBl getTasksManagerBl();
```

Comment:
Interface of ``PerunBl`` should also contain getter for non-BL version of ``TasksManager`` (like other managers do).